knora/dsplib/models/user.py

```diff
@@ -475,18 +476,17 @@ def fromJsonObj(cls, con: Connection, json_obj: Any):
         in_groups: set[str] = set()
         if json_obj.get('permissions') is not None and json_obj['permissions'].get('groupsPerProject') is not None:
```

Comment:
generally, `if x is not None` can normally be rewritten as `if not x` (but that doesn't need to be done in this PR... I know it's all over the place in dsp-tools)